package.json

```diff
@@ -351,6 +351,12 @@
           "type": "integer",
           "default": 5000,
           "markdownDescription": "The maximum number of outline symbols and folding regions computed (limited for performance reasons). Default is `5000`."
+        },
+        "xml.symbols.showReferencedGrammars": {
+          "type": "boolean",
+          "default": true,
+          "markdownDescription": "Show referenced grammars (Outline). Default is `true.",
```

Comment:
I think: "Show referenced grammars in the outline" would be a bit more clear

Reply:
fixed